M2/Macaulay2/packages/SumsOfSquares.m2

```diff
@@ -1436,6 +1436,7 @@ TEST /// --solveSOS
 
 --11
 TEST /// --lowerBound
+    setRandomSeed 123
```

Comment:
I still don't understand. Why would the hash codes be different on different builds?

Reply:
Some hash codes are generated from pointers to objects (e.g., hash codes for functions), so those will vary from build to build.  But I'm not sure if that's what's going on here or not.  It could be that the hash codes are the same for each build and CSDP is getting the same matrix each time, but CSDP itself is treating them differently.  Maybe their algorithm is probabilistic?  I'm not sure.